app/Http/Controllers/ProfileController.php

```diff
@@ -22,17 +23,29 @@ public function getProfileSettings()
     }
 
     /**
+     * Gets selected user's dashboard.
+     *
+     * @return Response
+     */
+    public function show($id)
+    {
+        $user     = User::with('projects')->find($id);
+
+        return view('projects.dashboard', compact('user', 'projects'));
```

Comment:
**Code Quality - Critical**

Why are you sending `user` and `projects` to the dashboard view? . I can't see any `$projects` variable anywhere.